demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1345,11 +1345,12 @@ def is_file_allowed_to_be_deleted(file_path):
 
         """
         file_path = str(file_path)
-        file_type = find_type(file_path)
+        file_dict = get_remote_file(file_path)
```

Comment:
```suggestion
        file_dict = get_remote_file(file_path, tag='master')
```
adding it explicitly helps future readers understand why this is called.